Connected System Plug-in (CSP) Examples/Google Drive (OAuth,  Sending Files and Multiple Operations)/src/main/java/com/mycorp/googledrive/templates/GoogleDriveListFilesIntegrationTemplate.java

```diff
@@ -0,0 +1,98 @@
+package com.mycorp.googledrive.templates;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+
+import com.appian.connectedsystems.simplified.sdk.SimpleIntegrationTemplate;
+import com.appian.connectedsystems.simplified.sdk.configuration.SimpleConfiguration;
+import com.appian.connectedsystems.templateframework.sdk.ExecutionContext;
+import com.appian.connectedsystems.templateframework.sdk.IntegrationResponse;
+import com.appian.connectedsystems.templateframework.sdk.TemplateId;
+import com.appian.connectedsystems.templateframework.sdk.configuration.PropertyPath;
+import com.appian.connectedsystems.templateframework.sdk.diagnostics.IntegrationDesignerDiagnostic;
+import com.google.api.client.googleapis.auth.oauth2.GoogleCredential;
+import com.google.api.client.googleapis.json.GoogleJsonResponseException;
+import com.google.api.client.http.javanet.NetHttpTransport;
+import com.google.api.client.json.jackson2.JacksonFactory;
+import com.google.api.services.drive.Drive;
+import com.google.api.services.drive.model.File;
+import com.google.api.services.drive.model.FileList;
+import com.google.common.base.Stopwatch;
+
+@TemplateId(name = "GoogleDriveListFilesIntegrationTemplate")
+public class GoogleDriveListFilesIntegrationTemplate extends SimpleIntegrationTemplate {
+  private static final String FOLDER_ID_KEY = "folderId";
+
+  @Override
+  protected SimpleConfiguration getConfiguration(
+      SimpleConfiguration integrationConfiguration,
+      SimpleConfiguration connectedSystemConfiguration,
+      PropertyPath updatedProperty,
+      ExecutionContext executionContext) {
+    return integrationConfiguration.setProperties(
+        textProperty(FOLDER_ID_KEY).label("Folder Id")
+            .instructionText("If left blank, it will list all existing files. Otherwise it will list the " +
```

Comment:
Do we want to localize examples? If we're giving them to customers, it may encourage localization if we localize our examples

Reply:
that's a good idea, but given that the existing operations in this CSP were not i18n I didn't worry too much